test/scalability/lib.py

```diff
@@ -42,6 +42,7 @@ def load_data_from_filesystem(
     workload_name: str,
 ) -> tuple[dict[str, pd.DataFrame], dict[str, pd.DataFrame]]:
     endpoint_names = paths.get_endpoint_names_from_results_dir()
+    endpoint_names.sort()
```

Comment:
Endpoint names would be `vX.Y.Z (SHA)`, right? If so, sorting them would be a problem -- for many benchmark runs, the X.Y.Z will be identical on both sides, with only the SHA changing . So as I commit more stuff, my SHA is going to be jumping from being before the SHA of the parent commit in sort order to being after it. So the boxplot of my PR will be either first or second as I push more into my branch?

Reply:
I understand that this could benefit from further refinement. I will note that down.
Until then, I still think that sorting is beneficial and better than the completely arbitrary order that I get from reading the folders in the directory.

Reply:
The order things were listed on the command line would be a good-enough, stable order. Feel free to open a follow-up ticket.